<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addVendorDialog</class>
 <widget class="QDialog" name="addVendorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>620</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add Vendor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="name_validation_label">
        <property name="text">
         <string>Validation label</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPlainTextEdit" name="companiesEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="requestorIdEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Customer ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Platform</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="platformEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="baseUrlEdit"/>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="non_Sushi_check_box">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPlainTextEdit" name="descriptionEdit"/>
      </item>
      <item row="10" column="0" alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Third party provider</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="url_validation_label">
        <property name="text">
         <string>Validation label</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="customerIdEdit"/>
      </item>
      <item row="9" column="0" alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Requestor ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Base URL</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Non-SUSHI Vendor</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="apiKeyEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>API Key</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="non_sushi_help_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/ui/resources/help_icon.png</normaloff>:/ui/resources/help_icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>baseUrlEdit</tabstop>
  <tabstop>customerIdEdit</tabstop>
  <tabstop>requestorIdEdit</tabstop>
  <tabstop>apiKeyEdit</tabstop>
  <tabstop>platformEdit</tabstop>
  <tabstop>non_Sushi_check_box</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>companiesEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
